Only in setiathome-7.08/AKv8/client: boinc_lockfile
Only in setiathome-7.08/AKv8/client: boinc_setiathome_0
Only in setiathome-7.08/AKv8/client: stderr.txt
Only in setiathome-7.08/AKv8/client: stderrgfx.txt
Only in setiathome-7.08/AP/client: .libs
Only in setiathome-7.08/AP/client: ap_7.01r1864_sse3_clGPU_x86_64-pc-linux-gnu
Only in setiathome-7.08/AP/client: ap_client
Binary files setiathome-7.08.orig/AP/client/ap_client-ap_client_main.o and setiathome-7.08/AP/client/ap_client-ap_client_main.o differ
Binary files setiathome-7.08.orig/AP/client/ap_client-ap_gfx_main.o and setiathome-7.08/AP/client/ap_client-ap_gfx_main.o differ
diff -ur -x '*.Po' -x requests -x config.log -x Makefile -x Makefile.in setiathome-7.08.orig/AP/client/ap_client_main.cpp setiathome-7.08/AP/client/ap_client_main.cpp
--- setiathome-7.08.orig/AP/client/ap_client_main.cpp	2015-12-06 20:46:29.158618083 -0800
+++ setiathome-7.08/AP/client/ap_client_main.cpp	2015-12-06 21:49:13.290192661 -0800
@@ -75,8 +75,8 @@
 #endif
 #ifdef BOINC_APP_GRAPHICS
 // #include "ap_graphics.h"   Not part of the main program any more
-//#include "ap_gfx_main.h"
-#include "ap_gfx_main.cpp"
+#include "ap_gfx_main.h"
+//#include "ap_gfx_main.cpp"
 #include "graphics2.h"
 #endif
 #if _GNU_SOURCE || __APPLE__ || __MINGW32__ || __FreeBSD__ //uje: gcc
@@ -92,7 +92,8 @@
 
 #include <time.h>
 
-
+REDUCED_ARRAY_GEN rarray;
+AP_GDATA* ap_gdata;
 long starttime = time(0);
 int lastcputicks = clock();
 float cputime = 0;
@@ -415,6 +416,7 @@
 double ffa_thresh_mult = 1.0;
 int selected_device = -1;
 double fpopsMult; //JWS: Should be added to state.
+inline bool nographics() { return nographics_flag; }
 
 APP_INIT_DATA app_init_data;
 
diff -ur -x '*.Po' -x requests -x config.log -x Makefile -x Makefile.in setiathome-7.08.orig/AP/client/ap_gfx_base.cpp setiathome-7.08/AP/client/ap_gfx_base.cpp
--- setiathome-7.08.orig/AP/client/ap_gfx_base.cpp	2015-12-06 20:46:29.161618202 -0800
+++ setiathome-7.08/AP/client/ap_gfx_base.cpp	2015-12-06 21:51:00.662979217 -0800
@@ -63,6 +63,7 @@
 #include "parse.h"
 #include "graphics_api.h"
 #include "ap_gfx_base.h"
+
 #include "gutil.h"
 #include "reduce.h"
 #include "graphics_data.h"
diff -ur -x '*.Po' -x requests -x config.log -x Makefile -x Makefile.in setiathome-7.08.orig/AP/client/ap_gfx_main.cpp setiathome-7.08/AP/client/ap_gfx_main.cpp
--- setiathome-7.08.orig/AP/client/ap_gfx_main.cpp	2015-12-06 20:46:29.167618439 -0800
+++ setiathome-7.08/AP/client/ap_gfx_main.cpp	2015-12-06 21:03:51.732716341 -0800
@@ -32,8 +32,6 @@
 #include "boinc_api.h"
 #include "graphics2.h"
 
-REDUCED_ARRAY_GEN rarray;
-AP_GDATA* ap_gdata;
 #ifdef BOINC_APP_GRAPHICS
 bool nographics_flag = false;
 #else
diff -ur -x '*.Po' -x requests -x config.log -x Makefile -x Makefile.in setiathome-7.08.orig/AP/client/ap_gfx_main.h setiathome-7.08/AP/client/ap_gfx_main.h
--- setiathome-7.08.orig/AP/client/ap_gfx_main.h	2015-12-06 20:46:29.161618202 -0800
+++ setiathome-7.08/AP/client/ap_gfx_main.h	2015-12-06 21:52:37.621403317 -0800
@@ -56,5 +56,4 @@
 extern REDUCED_ARRAY_GEN rarray;
 extern AP_GDATA* ap_gdata;
 extern bool nographics_flag;
-extern bool nographics() { return nographics_flag; }
 extern void ap_graphics_init(APP_INIT_DATA&);
Only in setiathome-7.08/AP/client: ap_graphics
Only in setiathome-7.08/AP/client: ap_graphics-ap_gfx_base.o
Only in setiathome-7.08/AP/client: ap_graphics-ap_graphics.o
Only in setiathome-7.08/AP/client: ap_graphics-graphics_main.o
diff -ur -x '*.Po' -x requests -x config.log -x Makefile -x Makefile.in setiathome-7.08.orig/AP/client/ap_graphics.cpp setiathome-7.08/AP/client/ap_graphics.cpp
--- setiathome-7.08.orig/AP/client/ap_graphics.cpp	2015-12-06 20:46:29.161618202 -0800
+++ setiathome-7.08/AP/client/ap_graphics.cpp	2015-12-06 21:52:22.660874693 -0800
@@ -89,6 +89,7 @@
 #elif defined(HAVE_GLUT_GLUT_H)
 #include <GLUT/glut.h>
 #endif
+
 #endif
 
 
Only in setiathome-7.08/AP/client: astropulse-7.01_x86_64-pc-linux-gnu.debug
Only in setiathome-7.08/AP/client: stderrgfx.txt
